app/lib/features/pins/providers/notifiers/edit_state_notifier.dart

```diff
@@ -76,9 +76,20 @@ class PinEditNotifier extends StateNotifier<PinEditState> {
         await updateBuilder.send();
       }
       await pin.refresh();
-      EasyLoading.showSuccess('Pin Updated Successfully');
+      if (!mounted) {
+        EasyLoading.dismiss();
+        return;
+      }
+      EasyLoading.showToast('Pin Updated Successfully');
     } catch (e) {
-      EasyLoading.showError('Error saving changes: ${e.toString()}');
+      if (!mounted) {
+        EasyLoading.dismiss();
+        return;
+      }
+      EasyLoading.showError(
+        'Error saving changes: ${e.toString()}',
+        duration: const Duration(seconds: 3),
+      );
```

Comment:
let's l10n these please

Reply:
@gnunicorn is thinking that arguably notifiers shouldn't be doing easyloading or translation stuff directly.
This part will be fixed at future PR.
Dropped comment [here](https://github.com/acterglobal/a3/pull/1580/commits/9da067c17b84ea59d58b98998eb93f414baf9eae).